make_profiler/report/index.html

```diff
@@ -0,0 +1,23 @@
+<html>
+
+<head>
+    <title>Kontur Make Profiler Report</title>
+    <link rel="icon" href="favicon.png" type="image/x-icon">
+    <link rel="stylesheet" href='report/style.css'>
+    <link href='https://fonts.googleapis.com/css?family=IBM Plex Sans Condensed' rel='stylesheet'>
+    <script src='report/script_u.js' type="text/javascript"> </script>
```

Comment:
`script_u.js`?

Reply:
just minified and uglified

Reply:
result of this command `uglifyjs --compress --mangle -- script.js `
so just content of script.js  inside

Reply:
wont use